### ISSUE TITLE
Add huge warning that installing this tweaks may break Plasma setup

### ISSUE THREAD

**Issue by kmaterka** [psifidotos/plasma-systray-latte-tweaks#5]
Hi,

please add clear and visible warning that installing this tweak may break Plasma setup. 
I spent several hours trying to investigate [Bug 423107](https://bugs.kde.org/show_bug.cgi?id=423107) :-(

Thanks!

**Comment by psifidotos** [comment]
Can you send a PR?

**Comment by michaldybczak** [comment]
The issue was, that the tweak was not updated on my system and that is why it conflicted with new systray. After update, systray icons are working, so the issue was not with the tweak itself, but the user - me and the lack of easy way to find and update it.

When I was looking for the widget to download, where we can see Update button, I didn't find this tweak when I searched for "latte".

Or, since latte is part of the KDE projects, including the tweak into the default systray would be nice. No need to update it separately and no need to port the tweak to new features? I know, I know, it probably won't happen from various reasons, but one could dream ;).

@kmaterka, the weird thing is that few others reported the same thing, although they had virtual systems, so I doubt they had this tweak. What are the odds? Anyway, I'll talk about it on Manjaro's forum and find out whether their problem has the same root. And sorry that I took so much of your time :(. I appreciate it and I really thought this is some general problem because others reported it as well.
I always try to avoid reporting bugs that are locally based, because I know how much time-consuming solving bugs can be, especially when the user or hardware is at fault (which is very often).
